plugins/modules/zos_operator.py

```diff
@@ -81,7 +104,19 @@
           "MV2C      2020039  04:29:57.59             IEE457I 04.29.57 UNIT STATUS 948  ",
           "         UNIT TYPE STATUS        VOLSER     VOLSTATE      SS                 ",
           "          0100 3277 OFFLINE                                 0                ",
-          "          0101 3277 OFFLINE                                 0                "
+          "          0101 3277 OFFLINE                                 0                ",
+          "---- if verbose security is true, additional output will resemble:",
```

Comment:
Thanks for the update, also given how it is approached where you use one `verbose` option or keep it granular `verbose`, `security`, `debug` , I would not have merged the doc to have only one example. RETURNS can support multiple example blocks in a array format. 

See [here](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#return-block) for this example how to return multiple samples:
```
    contains:
        missing:
            description: Packages that are missing from the system.
            returned: success
            type: list
            sample:
                - libmysqlclient-dev
                - libxml2-dev
```

Reply:
Also I don't think its exactly accurate anymore to rely on old doc "The response resulting from the execution of the operator command" , if I understand the code its either you get a RC from the result of a command or you get RC + all the added verbosity (debug, security, etc), nor does "The response resulting from the execution of the operator command" really describe what you are getting back anymore.

Reply:
Thank you for the updates.